test/sqllogictest/transform/literal_lifting.slt

```diff
@@ -847,19 +846,18 @@ Explained Query:
         Project (#0{a}) // { arity: 1 }
           Join on=(#0{a} = #1) type=differential // { arity: 2 }
             implementation
-              %1[#0]UK » %0:t[#0{a}]Kiif
+              %0:t[#0{a}]Kiif » %1[#0]Kiif
```

Comment:
Similarly unclear, but also a join involving a collection that is either errored or empty.